vital/types/metadata.cxx

```diff
@@ -158,11 +169,12 @@ metadata
 
 }
 
+// ----------------------------------------------------------------------------
 void
 metadata
-::add( metadata_item* item )
+::add( std::unique_ptr< metadata_item >&& item )
 {
-  this->m_metadata_map[item->tag()] = item_ptr(item);
+  this->m_metadata_map[ item->tag( )] = item_ptr{ item.release() };
```

Comment:
Oh, fun... this is broken, can you spot why? :wink:

I'm waiting on builds again, but hopefully this was the only issue. (Not counting the build errors in this version of the PR that I've already fixed.)